inference-engine/src/mkldnn_plugin/nodes/mkldnn_shuffle_channels_node.cpp

```diff
@@ -131,101 +107,123 @@ void MKLDNNShuffleChannelsNode::createPrimitive() {
     if (getSelectedPrimitiveDescriptor() == nullptr)
         THROW_SHCH_ERROR << "has unidentified preferable primitive descriptor";
 
-    const bool isBlocked = getParentEdgeAt(0)->getMemory().getDesc().hasLayoutType(LayoutType::nCsp8c) ||
-                           getParentEdgeAt(0)->getMemory().getDesc().hasLayoutType(LayoutType::nCsp16c);
+    const auto& memoryDesc = srcMemPtr->getDesc();
+    attrs.batchRank = attrs.axis;
```

Comment:
looks like `batchRank` value == `axis` value
can we remote `batchRank` from structure?

Reply:
I guess it was for a better understanding of what this code does. I think it would be better to leave it. But I moved it to the executor ctor and removed it as a struct field